### PR TITLE
Move .xcworkspace / bridging header task warning to pod init error

### LINKED ISSUE #554: Move .xcworkspace to "pod init" error message to reduce spew
Now that j2objcXcode runs much more frequently, these notes appear more often.

### PR DESCRIPTION
**Pull request by brunobowden**
- Move .xcworkspace message from task warning to pod init error
- Fixes #554

### CODE REVIEW

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy [hunk 1]
```diff
@@ -174,14 +174,20 @@ class XcodeTask extends DefaultTask {
                     "To fix this:\n" +
                     "\n" +
                     "1) Set xcodeProjectDir to the directory containing 'IOS-APP.xcodeproj':\n" +
-                    "    curent value: ${getXcodeProjectDir()}\n" +
-                    "    resolves to: $xcodeAbsPath\n" +
+                    "    current value: ${getXcodeProjectDir()}\n" +
+                    "    resolving to: $xcodeAbsPath\n" +
```
Comment (advayDev1) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy]: how about just 'absolute path:'
i think that is more understandable than 'resolution'

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy [hunk 2]
```diff
@@ -174,14 +174,20 @@ class XcodeTask extends DefaultTask {
...
                     "\n" +
                     "2) Within that directory, create the Podfile with:\n" +
                     "    (cd $xcodeAbsPath && pod init)\n" +
                     "\n" +
                     "If the pod command isn't found, then install CocoaPods:\n" +
-                    "    sudo gem install cocoapods"
+                    "    sudo gem install cocoapods\n" +
+                    "\n" +
+                    "NOTE: after building, open the '.xcworkspace' file in Xcode. Opening '.xcodeproj' will fail.\n" +
```
Comment (advayDev1) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy]: Capitalize After

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy [hunk 3]
```diff
@@ -174,14 +174,20 @@ class XcodeTask extends DefaultTask {
...
+                    "NOTE: when working with Swift, setup your bridging header:\n" +
```
Comment (advayDev1) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy]: Capitalize When

Comment (advayDev1) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy]: also ad another newline before this note so 2 NOTEs are apparent

**Comment by advayDev1** [comment]
lgtm + comment

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy (hunk 1 above)
Comment (brunobowden) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy]: Even better, it aligns too...

```
    current value: ${getXcodeProjectDir()}
    absolute path: $xcodeAbsPath
```

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy (hunk 2 above)
Comment (brunobowden) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy]: done

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy (hunk 3 above)
Comment (brunobowden) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/XcodeTask.groovy]: Both done. Good idea on the separation.

**Comment by brunobowden** [comment]
Build failed due to Gradle download. Merging anyway.

**Pull request merged**